### DISCUSSION TITLE
Customer search by Phone Number

### DISCUSSION THREAD

**Discussion by zjoe29** [Blair2004/NexoPOS#879] [Ideas]
HI team,
Maybe team can add customer search by Phone Number or some Barcode, ID,...
Because when we search by Name, it's difficult to find customer with the same name ( which is popular in my country ).

**Comment by Blair2004** [comment]
Hi, 
Barcode ? Customers doesn't have a barcode on NexoPOS 4x, i don't think for now it's relevant.
However for the phone number, it should be possible soon.
Thanks for the suggestion anyway.

**Reply by zjoe29** [reply]
Hi, Just my suggestion about Barcode, The real problem is : When the cashier search by name, many customer have  a same name, so cashier dont know which one is right.
So can you update allow cashier ( when POS Open ) type phone number to find customer? (Because Phone number is Unique on every customer)

**Reply by Blair2004** [reply]
Yes we'll consider adding the phone number
